src/main/java/core/basesyntax/FigureBehaviour.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureBehaviour {
+    double getArea();
+
+}
```

Comment:
i think this is not the best naming https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-put-all-behaviour-into-a-single-interface-if-the-methods-are-conceptually-different-from-each-other